src/shared/lib/time.ts

```diff
@@ -1,24 +1,23 @@
 import { format } from 'date-fns/format';
+import { useI18n } from 'vue-i18n';
+
+export function getTimeAgo(now: number, dateString: string | Date) {
+  const { t } = useI18n();
```

Comment:
`useI18n` is a composable that is meant to be called _once_ during `setup`.

So, this properly should be either 

- `useTimeAgo(date: RefOrGetter<Date>): Ref<string>`, or
- `getTimeAgo(now: number, date: Date, t: ...): string` 

And I think it's better to be a composable which "cleverly" tracks current time and updates the value reactively with a reactive timer, all inside.